Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/AdminDistTransforms.java

```diff
@@ -178,7 +179,11 @@ protected String getHomeCommunityFromMapping() {
     }
 
     private String getRelatesTo(AssertionType assertion) {
-        return NullChecker.isNotNullish(assertion.getRelatesToList()) ? assertion.getRelatesToList().get(0) : null;
+        return NullChecker.isNotNullish(assertion.getRelatesToList()) ? getRelatesToValue(assertion.getRelatesToList())
+            : null;
     }
 
+    private String getRelatesToValue(List<String> relatesTo) {
+        return NullChecker.isNotNullish(relatesTo.get(0)) ? relatesTo.get(0) : null;
```

Comment:
If an empty relatesTo is passed either in assertion block or in SOAP header, both Nwhin and db will log it as null. For reference please look at the below QA finding:
For PD Def. Response in unsecured message, if an empty <relatesTo> in Soap header block is passed, both outbound and inbound entries storing null in auditrepository table.
 But, if an empty<relatesTo> is passed in assertion block, then outbound have "empty" and inbound entry have "null" in auditrepository table.